bindata/manifests/metrics-exporter/metrics-config-map.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriov-network-metrics-exporter-config
+  namespace: {{.Namespace}}
+data:
+  drivers.yaml: |-
+    drivers:
+      - name: ice
+        version: 1.9.11
+      - name: mlx5_core
+        version: 5.15.0-53-generic
```

Comment:
How these settings work? Can we rely on these hardcoded values for all environments?

Reply:
agree @zeeke will it be possible to allow the exporter to work without this file?
I am getting the feeling that it will be super hard to maintain this one updated

Reply:
sriov-network-metrics-exporter needs those settings; they have been introduced [here](https://github.com/k8snetworkplumbingwg/sriov-network-metrics-exporter/pull/22/commits/9ce872aa1239f076e1031f7abd97bbf13b008174#diff-6e418e33107e875bb2f5b586d09d6a7359369cf4753e496ffaa875562b33b32b). The good part is that they declare the _minumim_ supported driver version. There should be no need to update them in the future unless new drivers are released from scratch.

@Eoghan1232 do you think we can make these settings non - mandatory?

Reply:
Hi @zeeke , Yeh I see no issues with making these settings non mandatory.
These values could be built in, and this could be an optional override of the minimum if need.
wdyt?

Reply:
https://github.com/k8snetworkplumbingwg/sriov-network-metrics-exporter/pull/35

Reply:
I updated the configuration to:
```yaml
  drivers.yaml: |-
    drivers:
      - name: ice
        version: 0.0.0
      - name: mlx5_core
        version: 0.0.0
```

This allows gathering metrics from any devices bound to the `mlx5_core` and `ice` driver, whichever version they are. This solves the problem of finding a good minimum version that works for both in-tree and out-of-tree drivers.

The discussion about how we can make this ConfigMap better and more maintainable continues here:
- https://github.com/k8snetworkplumbingwg/sriov-network-metrics-exporter/pull/35

@SchSeba @e0ne @Eoghan1232 WDYT?

Reply:
the configmap has been removed, as of:
- https://github.com/k8snetworkplumbingwg/sriov-network-metrics-exporter/pull/37